fabtests/component/dmabuf-rdma/fi-mr-reg-xe.c

```diff
@@ -171,10 +171,43 @@ void reg_mr(void)
 	return;
 }
 
+void reg_dmabuf_mr(void)
+{
+	struct fi_mr_dmabuf dmabuf = {
+		.fd = xe_get_buf_fd(buf),
+		.offset = 0,
+		.len = buf_size,
+	};
+	struct fi_mr_attr mr_attr = {
+		.dmabuf = &dmabuf,
+		.access = FI_REMOTE_READ | FI_REMOTE_WRITE,
+		.requested_key = 2,
+	};
+
+	CHECK_ERROR(fi_mr_regattr(domain, &mr_attr, FI_MR_DMABUF, &dmabuf_mr));
+
+	if (fi->domain_attr->mr_mode & FI_MR_ENDPOINT) {
+		CHECK_ERROR(fi_mr_bind(dmabuf_mr, &ep->fid, 0));
+		CHECK_ERROR(fi_mr_enable(dmabuf_mr));
+	}
+
+	printf("mr %p, buf %p, rkey 0x%lx, len %zd\n",
+		mr, buf, fi_mr_key(mr), buf_size);
```

Comment:
should print out `dmabuf_mr` instead of `mr`, same for the mr key.

Reply:
Thanks - updated.  And thanks for testing. :)